source/UF/Truncations.lagda

```diff
@@ -150,6 +150,23 @@ groupoid).
 
 \end{code}
 
+We show that the existence of propositional truncation follows from the existence
+of general truncations. Notice this implication manifests as a function between
+record types.
+
+\begin{code}
+
+ prop-trunc-follows-from-gen-trunc : H-level-truncations-exist
+                                   → propositional-truncations-exist
+ prop-trunc-follows-from-gen-trunc te = record
```

Comment:
You're not using the assumption `te` here. Rather you're within the scope of the original `H-level-truncations-exist` record.
I would also rename this to `H-level-truncations-give-propositional-truncations`.

Reply:
You can simply exit the scope by indenting one level less.

Reply:
Oh right. So should I do this outside of the record?

Reply:
You can also do it within the record, but then its type is `propositional-truncations-exist`, so I think doing it outside the record is clearer.

Reply:
Within the record is like going outside and then doing `open H-level-truncations-exist te`.

Reply:
> You can simply exit the scope by indenting one level less.

When I indent it gives me an error on the code that follows that is supposed to be back within the scope of the record.

Reply:
Like I said, you have to open the record with your assumption.

Reply:
It works just as with modules.

Reply:
I understand that I exited the scope then opened the record with my assumption using a where statement. The problem is that indenting below doesn't put me back in the scope right? It would be nice to prove this and then use it in the file to avoid an pt assumption at all.

Reply:
I don't understand `avoid any pt assumption at all`? But if you want the existence of propositional truncations to become available as soon as you have general truncations, then you can prove this fact within the scope of general truncations record, but you just simplify change its type to `propositional-truncations-exist`.

Does that help?

Reply:
Ahh I guess I see your point. Let me push what I decided to go with and see what you think.